UnityProject/Assets/Editor/LevelBuilder/GUI/Menus/GeneratorMenu.cs

```diff
@@ -57,40 +61,40 @@ private void GeneratorGUI(Type generatorType, string buttonName, GenerationData
 
 		GUILayout.BeginHorizontal();
 
-		// A dropdown list which displays all the PNG files that can be used as spites 
-		// See AssetController for 
+		// A dropdown list which displays all the PNG files that can be used as spites
+		// See AssetController for
 		IList<string> spriteList = AssetFetcher.GetSprites();
 		spriteList.Insert(0, "Default Sprite");
 
 		string[] sprites = spriteList.ToArray<string>();
-		GUILayout.Label("Sprite:"); 
+		GUILayout.Label("Sprite:");
 		data.idx = EditorGUILayout.Popup(data.idx, sprites);
 
 		GUILayout.EndHorizontal ();
 
-		if (GUILayout.Button (new GUIContent (buttonName))) 
+		if (GUILayout.Button (new GUIContent (buttonName)))
 		{
-			// We use the Builder pattern so that the GUI logic can get updated easily 
+			// We use the Builder pattern so that the GUI logic can get updated easily
 			builder = builder.ByCoord(0.0f, 0.0f);
-			if (data.idx != 0) 
+			if (data.idx != 0)
 				builder = builder.ByPath (sprites [data.idx]);
-		
-			if (data.width != "") 
+
+			if (data.width != "")
```

Comment:
how come width and heights are strings?

Reply:
I looked for a while now, in both `GeneratorMenu` and `SpriteGeneratorBuilder` and can't find an answer for this. Maybe @danalex97?

Reply:
I think I kept using [text fields](https://docs.unity3d.com/ScriptReference/EditorGUILayout.TextField.html) while prototyping and forgot to replace those with [float fields](https://docs.unity3d.com/ScriptReference/EditorGUI.FloatField.html) or [int fields](https://docs.unity3d.com/ScriptReference/EditorGUI.IntField.html). We can open an issue to making all the GUI elements more consistent.

Reply:
Issue is open. @CelineBoudier - do you want me to fix this now for this PR or do it later on when I go through the issues?

Reply:
It's ok to look at it later as technically this works I guess